### DISCUSSION TITLE
Github does not accept domen simplelogin.com to registration.  Is this bug or no?

### DISCUSSION THREAD

**Discussion by SL20230318** [simple-login/app#1647] [Q&A]
Github does not accept domen simplelogin.com to registration.  Is this bug or no?

**Comment by sergiorgiraldo** [comment]


**Reply by SL20230318** [reply]
> It is on their side, some companies do not allow 'anonymous' emails

Thank you.  But if there are many (in percents) such companies, what is the point in «anonymous» email addresses?

**Reply by SL20230318** [reply]
Thank you.

**Comment by nguyenkims** [comment]
We are aware of the issue and will get in touch with Github to resolve it.

**Reply by Sntx626** [reply]
What's the status on this?
